src/sentry/static/sentry/app/views/settings/components/tag.jsx

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
 import styled from 'react-emotion';
 import PropTypes from 'prop-types';
+import {Box} from 'grid-emotion';
 
-const TagTextStyled = styled('span')`
+const getMarginLeft = p => {
+  if (!p.inline) return '';
+  return `margin-left: ${p.size == 'small' ? '0.25em' : '0.5em'};`;
+};
+
+const TagTextStyled = styled(Box)`
   display: inline;
-  padding: 0.4em 0.6em;
-  font-size: 75%;
-  font-weight: bold;
+  padding: ${p => (p.size == 'small' ? '0.1em 0.4em 0.2em' : '0.35em 0.8em 0.4em')};
+  ${p =>
+    p.border
+      ? `border: 1px solid ${p.priority
+          ? p.theme.alert[p.priority].border
+          : p.theme.gray1};`
+      : ''} font-size: 75%;
```

Comment:
Should this fontsize be hanging out here? Seems like we need a semicolon after this.